hypervisor/arch/x86/cpu_state_tbl.c

```diff
@@ -62,6 +79,10 @@ struct cpu_state_table {
 		{ARRAY_SIZE(px_a3960), px_a3960,
 		 ARRAY_SIZE(cx_a3960), cx_a3960}
 	},
+	{"Intel(R) Atom(TM) Processor A3950 @ 1.60GHz",
```

Comment:
Then you need to add 1.9G case too, right?

Reply:
1.9G is already in, it is the first SoC we support when enable Pstate.